package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "discord-hangman",
-  "version": "1.2.2",
-  "description": "🎲 Complete framework to facilitate the creation of hangman games using discord.js",
+  "version": "1.3.0",
+  "description": "🎲 Complete framework to facilitate the creation of hangman games using discord.js support interaction on Discord.js V13",
```

Comment:
The sentence doesn't mean anything anymore haha! The basic description is enough in my opinion.

Reply:
Corriger